atomic_reactor/plugins/post_import_image.py

```diff
@@ -83,15 +91,16 @@ def run(self):
             osbs.create_image_stream(self.imagestream, self.docker_image_repo,
                                      **kwargs)
         else:
-            self.log.info("Importing tags for %s", self.imagestream)
-            retry_attempts = 3
-            while True:
-                result = osbs.import_image(self.imagestream, **kwargs)
-                if result:
-                    break
-
-                if retry_attempts > 0:
-                    retry_attempts -= 1
-                    self.log.info("no new tags, will retry after %d seconds",
-                                  self.retry_delay)
-                    sleep(self.retry_delay)
+            self.log.info("Importing new tags for %s", self.imagestream)
+
+            attempts = 0
+            while not osbs.import_image(self.imagestream):
```

Comment:
More like a style comment, but I think a for-loop is probably cleaner here. Feel free to ignore this comment :)